scripts/generate.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/env python
+
+"""Generate build and run scripts for Unikraft application.
+
+Use KrafKit and Make for building. Use QEMU, Firecracker and Krafkit
+for running.
+"""
+
+import sys
+import os
+import stat
+import yaml
+
+TEMPLATE_BUILD_MAKE = """#!/bin/sh
+
+make distclean
+UK_DEFCONFIG=$(pwd)/defconfigs/{} make defconfig
+make prepare
+make -j $(nproc)
+"""
+
+TEMPLATE_BUILD_KRAFT = """#!/bin/sh
+
+rm -fr .unikraft
+rm -f .config.*
+kraft build --log-level debug --log-type basic --target {} --plat {}
```

Comment:
I get this error if I try to run helloworld-c without sudo:
` E  failed to create volume: could not open machine store: Invalid Dir: "/var/kraftkit/volumev1alpha1" error: stat /var/kraftkit/volumev1alpha1: permission denied`

It works fine if I run it with sudo, it might something on my side since nginx works fine either way.

Reply:
Probably some KraftKit-related issue.